requirements.txt

```diff
@@ -12,6 +12,8 @@ starlette==0.26.1
 uvicorn==0.21.1
 openpyxl==3.1.2
 mysql-statement-builder==0.*
+python-magic==0.4.27
```

Comment:
I could not find where this req is being used. Do we need it?

Reply:
This is used in cvs dsm file upload for checking if filetype matches the extension. If it is deemed unneccessary, we can remove it.

Reply:
If it is only used to check the extension of the file, then we should stick to the functionality that is already in the os and path libs. If it actually opens the file and reads its headers to check if it is the correct file type, then we can reconsider.

Reply:
Python-magic is an interface for linux's own libmagic library. It looks at the contents of the file and tries to guess what kind of file it is. So what I wanted to do was to double check that the extension matches the contents of the file since it is quite easy to just rename a file and change the extension.

Reply:
That sounds appropriate, thanks for the info. We will in that case keep it.